src/plugins/lambda-entry.ts

```diff
@@ -41,12 +41,6 @@ export const lambdaEntryPlugin: (features: {
         compiler.onEnd(async (result) => {
             await Promise.all(
                 result.outputFiles?.map(async (file) => {
-                    if (esm && file.path.endsWith('.js') && file.text.includes('__require')) {
-                        file.contents = Buffer.concat([
-                            Buffer.from(`const require = (await import('node:module')).createRequire(import.meta.url);`),
-                            file.contents,
-                        ])
-                    }
```

Comment:
this removal needs to be reverted, this is required to force-bundle certain npm packages